docs/API.md

```diff
@@ -0,0 +1,50 @@
+
+### Root - AsyncAPIDocument
+- `AsyncAPIDocument.hasContentType('<content type>')` : boolean (outcome of comment(s): [799481319](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-799481319))
+- `AsyncAPIDocument.channels()` : Channel[] (outcome of comment(s): [800282407](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-800282407), [800935961](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-800935961))
+- **(HELP NEEDED)** `AsyncAPIDocument.channelsSubscribes('<role>')` : Channel[] (outcome of comment(s): [800282407](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-800282407), [800935961](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-800935961))
+- **(HELP NEEDED)** `AsyncAPIDocument.channelsPublishes('<role>')` : Channel[] (outcome of comment(s): [800282407](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-800282407), [800935961](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-800935961))
+- **(HELP NEEDED)** `AsyncAPIDocument.isSubscribingToChannels('<role>')` : boolean (outcome of comment(s): [799481319](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-799481319))
+- **(HELP NEEDED)** `AsyncAPIDocument.isPublishingToChannels('<role>')` : boolean (outcome of comment(s): [799481319](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-799481319))
```

Comment:
I don't understand the intent behind these methods. If the intent is to understand if we're subscribing to a given channel(s), where's the channel?

I'd make it like this:

```suggestion
- **(HELP NEEDED)** `AsyncAPIDocument.isSubscribingToChannel('<channel name>')` : boolean (outcome of comment(s): [799481319](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-799481319))
- **(HELP NEEDED)** `AsyncAPIDocument.canClientSubscribeToChannel('<channel name>')` : boolean (outcome of comment(s): [799481319](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-799481319))
- **(HELP NEEDED)** `AsyncAPIDocument.isPublishingToChannel('<channel name>')` : boolean (outcome of comment(s): [799481319](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-799481319))
- **(HELP NEEDED)** `AsyncAPIDocument.canClientPublishToChannel('<channel name>')` : boolean (outcome of comment(s): [799481319](https://github.com/asyncapi/shape-up-process/issues/84#issuecomment-799481319))
```

Again, as before, the role is implicit in the method name.

Reply:
The API is not seen from the outside, but from a user who uses the API with different use-cases.

Reply:
We didn't need to check for a specific channel, but in general 😄

Reply:
I don't get it. What was the meaning of `isSubscribingToChannels('<role>')` then? Can you provide a description as if it was for documentation?

Reply:
Now that I revise it the use-case is wrong or rather not existing since the use-case uses `Channel.isSubscribing('<role>')` instead 🤔 

Gonna remove it.

Reply:
Removing it since we are only following use-cases at the moment 🙂